### PR TITLE
Refactor person to patient

### LINKED ISSUE #69: Refactor Person to Patient

### PR DESCRIPTION
**Pull request by xz0127**
Refactor all usages and text occurrences of person(s) / PERSON(S) / Person(s) into patient(s) / PATIENT(S) / Patient(s).

Update DeveloperGuide by changing text occurrences of person into patient

### CODE REVIEW

**Review by Avalionnet — APPROVED** [review]
Excellent work!! LGTM :)

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -239,7 +239,7 @@ Namise is a hard working nurse working at a popular dental clinic situated in to
 
   
 
-Being a tech-savvy person armed with a commendable experience in unix, Namise prefers to scribble down appointment schedules on paper while on call with his patients to maximise efficiency. This task is further exacerbated with the need to transfer these notes into an excel table manually later in the day.
+Being a tech-savvy patient armed with a commendable experience in unix, Namise prefers to scribble down appointment schedules on paper while on call with his patients to maximise efficiency. This task is further exacerbated with the need to transfer these notes into an excel table manually later in the day.
```
Comment (JinHao-L) [inline docs/DeveloperGuide.md]: I think this should be person or nurse instead

**Review by JinHao-L — CHANGES_REQUESTED** [review]
Other than the minor edit in the comment, the rest LGTM!

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (xz0127) [reply docs/DeveloperGuide.md]: Oh right i missed that part. Thanks for pointing out! 👍

**Review by JinHao-L — APPROVED** [review]
LGTM 👍

**Pull request merged**